serverless/verify-backup/scripts/deploy.sh

```diff
@@ -15,9 +15,11 @@ echo 'Deploying build to Cloud Run...'
 # --set-env-vars `grep -v '^#' .env | awk -v ORS=, 'NF { print $1 }'`pulls env vars from .env file and formats as args
 gcloud run deploy $GCLOUD_RUN_SERVICE_NAME \
   --region asia-southeast1 \
-  --no-allow-unauthenticated  \
+  --no-allow-unauthenticated \
   --image gcr.io/postmangovsg/verify-backup \
   --service-account $GCLOUD_SERVICE_ACCOUNT \
   --platform managed \
   --set-env-vars `grep -v '^#' .env | awk -v ORS=, 'NF { print $1 }'` \
-  --memory 4G
\ No newline at end of file
+  --timeout 5m \
```

Comment:
Perhaps we should [cap the maximum container](https://cloud.google.com/run/docs/configuring/max-instances#command-line) instance by setting `--max-instances 1` also? When testing it out, I noticed that when the response takes awhile to return, Cloud Run starts up another instance. Might be good to explicitly hard limit this to prevent a runaway auto-scale since the default is 1000 instance.

Reply:
Yes that'll be a good precaution to put in, will set that

Reply:
After testing, fixing the max instances at 1 increases the time taken for the request to complete. Its ~5.5mins now, so I think we can raise the request timeout from 5 mins to 8 mins?